HedvigInsurance/web-onboarding
# GRW-776 Fix issue where startDate would be set to today when updating quote data

## What?

The start date goes from null (when my old insurance ends) to “today” after phone number/SSN is submitted.

How to reproduce:
* Create a quote for NO home while stating that you have a current insurer
* > The offer page should state that the startDate for the new insurance should be when the old one is cancelled
* Go to the checkout and add phone number and SSN
* Expected behavior: > The startDate is not affected
* Actual behavior: > The startDate becomes "Today"

**Ticket(s): [GRW-776]**

## Screenshots / recordings 

https://user-images.githubusercontent.com/89857278/152787452-7652fb08-aff9-4a3d-9261-76af5c0a4ddc.mov

### [Review app](https://web-onboardi-grw-776-qc-ttoqjn.herokuapp.com/no-en/new-member/home/start)




[GRW-776]: https://hedvig.atlassian.net/browse/GRW-776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
